crates/store/src/state.rs

```diff
@@ -1054,7 +1071,21 @@ async fn load_mmr(db: &mut Db) -> Result<Mmr, StateInitializationError> {
 #[instrument(level = "debug", target = COMPONENT, skip_all)]
 async fn load_accounts(db: &mut Db) -> Result<AccountTree, StateInitializationError> {
     let account_data = db.select_all_account_commitments().await?.into_iter().collect::<Vec<_>>();
+    let len = account_data.len();
+
+    let now = Instant::now();
+
+    let account_tree = AccountTree::with_entries(account_data)
+        .map_err(StateInitializationError::FailedToCreateAccountsTree)?;
+
+    let elapsed = now.elapsed().as_secs();
+
+    info!(
+        num_of_leaves = len,
+        tree_construction = elapsed,
+        COMPONENT,
+        "Loaded account tree"
+    );
```

Comment:
Do we want to keep this? `elapsed` is also unused I think so it makes me think this was used while developing only?

Reply:
We are using the `elapsed` only in the `info!`. I don't have any strong preference about this, I'll remove it now but if anyone prefer to have it back I can rollback the change.